index.html

```diff
@@ -2403,6 +2408,98 @@ <h3>
           can get their games and apps rated with IARC</a>.
         </p>
       </section>
+      <section>
+        <h3>
+          <code>shortcuts</code> member
+        </h3>
+        <p>
+          The <dfn>shortcuts</dfn> member is an <a>array</a> of
+          <a>ShortcutItem</a>s that provide access to key tasks within a web
+          application. For example, they can be used to link directly to a
+          user's timeline within a social media application or their recent
+          orders in an e-commerce context.
+        </p>
+        <p>
+          A developer may provide as many shortcuts as they like, but the user
+          agent and operating system will decide how many are presented to the
+          user. For this reason, it is recommended that developers order their
```

Comment:
This sentence is non-normative and should be in a note. Also you can't use the keyword "recommended" when giving instructions to the developer (since this spec governs how UAs work, not how developers should behave). Reword as "Developers are encouraged to order their ...".